src/Google/Ads/GoogleAds/Lib/ConfigurationTrait.php

```diff
@@ -207,4 +210,14 @@ public function getGrpcInterceptors()
     {
         return $this->grpcInterceptors;
     }
+
+    /**
+     * Gets the REST HTTP handler.
+     *
+     * @return Guzzle6HttpHandler|Guzzle7HttpHandler
+     */
+    public function getRestHttpHandler()
```

Comment:
The term `Rest` already implies `Http`, and. vice-versa, so why not call this simply `getHttpHandler`?

Reply:
Done.